assets/evaluation_results/boolq_mistral_7b_instruct_v01_question_answering/spec.yaml

```diff
@@ -0,0 +1,24 @@
+type: evaluationresult
+name: boolq_mistral_7b_instruct_v01_question_answering
+version: 1.0.0
+display_name: boolq_mistral_7b_instruct_v01_question_answering
+description: mistralai-mistral-7b-instruct-v01 run for boolq dataset
+dataset_family: boolq
+dataset_name: boolq
+
+model_name: mistralai-mistral-7b-instruct-v01
+model_version: "1"
+model_asset_id: azureml://registries/azureml/models/mistralai-Mistral-7B-Instruct-v01/versions/1
+
+relationships:
+  - relationshipType: Source
+    assetId: azureml://registries/azureml/models/mistralai-Mistral-7B-Instruct-v01/versions/1
+
+tags:
+  task: question-answering
+  accuracy_metric_name: exact_match
+
+metrics:
+  accuracy: 0.777
+
+path: ./job_files
```

Comment:
looks like the mfe_json is missing for all results.

Reply:
These were disabled by Yunsong. We need to re-enable them in future. But for now, they are not needed as we don't do job regeneration.